spec/test-outputs/www-integration.out.vcl

```diff
@@ -1,23 +1,14 @@
 backend F_origin {
     .connect_timeout = 5s;
-    .dynamic = true;
     .port = "443";
-    .host = "foo";
+    .host = "127.0.0.1";
     .first_byte_timeout = 15s;
     .max_connections = 200;
     .between_bytes_timeout = 10s;
-    .share_key = "123";
-
-    .ssl = true;
-    .ssl_check_cert = always;
-    .min_tls_version = "1.2";
-    .ssl_cert_hostname = "foo";
-    .ssl_sni_hostname = "foo";
```

Comment:
These seem to be Fastly-specific. One solution would be to wrap lines like this in an `<% unless testing %>` block so that the VCL will compile for varnishtest. However, they would need to be manually tested, perhaps with [Fastly Fiddle](https://fiddle.fastlydemo.net/).